acados/ocp_nlp/ocp_nlp_sqp_rti.c

```diff
@@ -1260,6 +1259,7 @@ int ocp_nlp_sqp_rti(void *config_, void *dims_, void *nlp_in_, void *nlp_out_,
     for (ii = 0; ii <= N; ii++)
     {
         config->constraints[ii]->memory_set_ux_ptr(nlp_out->ux+ii, mem->constraints[ii]);
+		config->constraints[ii]->memory_set_dzdux_tran_ptr(mem->dzduxt+ii, mem->constraints[ii]);
```

Comment:
`memory_set_z_alg_ptr` is missing I guess

Reply:
addressed here 55fb99bc14